school/diary/tests.py

```diff
@@ -1,9 +1,106 @@
 from django.test import TestCase
 import pytest
+from django.urls import reverse
+from pytest_django.asserts import assertQuerysetEqual
 
+from .models import FootballPlayer, Achievement
 
-@pytest.mark.urls()
-def test_index_redirects_to_football_player(client):
-    response = client.get('/')
+
+# check is redirection works correctly
+@pytest.mark.urls('school.urls')
+def test_is_exist_redirect(client):
+    """
+    testing status_code matching
+    :param client:
+    :return:
+    """
+    response = client.get('/is_exist_check/')
```

Comment:
it would be better to use the `reverse` function to build URLs even in the tests.